src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+ul,
+li,
+body,
+nav {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-left: 50px;
+  padding-top: 10px;
+  padding-bottom: 10px;
```

Comment:
You can use shorthand properties instead
https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax